ui/pages/add-snap-account/snap-account-detail-page/snap-account-detail-page.test.tsx

```diff
@@ -0,0 +1,199 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import messages from '../../../../app/_locales/en/messages.json';
+import { KEY_MANAGEMENT_SNAPS } from '../../../../app/scripts/controllers/permissions/snaps/keyManagementSnaps';
+import mockState from '../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import SnapAccountDetailPage from '.';
+
+const snap = Object.values(KEY_MANAGEMENT_SNAPS)[0];
+
+// const mockInstallSnapFromSnapAccounts = jest.fn();
+
+// jest.mock('../../../store/actions.ts', () => ({
+//   installSnapFromSnapAccounts: () => mockInstallSnapFromSnapAccounts,
+// }));
```

Comment:
Remove

Reply:
removed in 063bb1866f84827894a49975dca7979347efdd97